packages/math/init.lua

```diff
@@ -344,6 +359,18 @@ Finally, here is a little secret. This notation:
 
 \noindent In other words, the notation using \code{&} and \code{\\\\} is only a syntactic sugar for a two-dimensional array constructed with braces.
 
+\paragraph{Numbered equations}
+Equations can be numbered in display mode adding the options \autodoc:parameter{numbered} and \autodoc:parameter{counter}.
+When \autodoc:parameter{numbered=true}, the equations are numbered using a default "equation" counter.
+A different counter may be set by using the option \autodoc:parameter{counter=id}, and this setting will also enable numbering. 
+The default numbering format is "(xx)", but this style may be overriden by defining a custom \autodoc:command{\math:numberingstyle} function, e.g.
```

Comment:
"(xx)" --> I would suggest \autodoc:example{(xx)}` so that it is typeset in the example font (Cormorant Infant currently) as other examples of "rendered content". We could avoid the quote marks here, then: no need for double marking! (But if we want to keep them, then use typographical quote marks.